### ISSUE TITLE
ondragstart and ondragend error

### ISSUE THREAD

**Issue by brunokunace** [phphe/he-tree-vue#1]
When set ondragstart and ondragerror props, receive error

```
[ERROR|/usr/local] {"text":"http://localhost:8080/_nuxt/2.js:5036_Uncaught TypeError: Cannot read property 'slice' of undefined_52_TypeError: Cannot read property 'slice' of undefined","level":"ERROR"} 

Uncaught TypeError: Cannot read property 'slice' of undefined
    at VueComponent.executeHook (2.js:5036)
    at Object.ondrag (2.js:1497)
    at Object.drag (2.js:662)
    at drag (2.js:1952)
    at HTMLDocument.moving (2.js:2006)
    at HTMLDocument.wrapper (2.js:1736)
```

eachDraggable and eachDroppable work fine, but i need to test when drag end, because i dont put "folder" inside another, only "files"

**Comment by phphe** [comment]
Thanks for issue report. Fixed in 1.0.1

**Comment by phphe** [comment]
Fixed
